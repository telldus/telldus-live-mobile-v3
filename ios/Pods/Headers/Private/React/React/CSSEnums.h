../../../../../../node_modules/react-native/React/CSSLayout/CSSEnums.h