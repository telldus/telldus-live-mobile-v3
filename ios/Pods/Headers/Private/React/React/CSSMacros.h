../../../../../../node_modules/react-native/React/CSSLayout/CSSMacros.h