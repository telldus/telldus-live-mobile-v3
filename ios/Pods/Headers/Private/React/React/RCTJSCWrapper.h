../../../../../../node_modules/react-native/React/Executors/RCTJSCWrapper.h