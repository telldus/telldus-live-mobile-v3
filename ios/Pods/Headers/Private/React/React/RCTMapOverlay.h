../../../../../../node_modules/react-native/React/Views/RCTMapOverlay.h