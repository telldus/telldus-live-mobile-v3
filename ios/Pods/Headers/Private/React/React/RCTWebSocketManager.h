../../../../../../node_modules/react-native/Libraries/WebSocket/RCTWebSocketManager.h