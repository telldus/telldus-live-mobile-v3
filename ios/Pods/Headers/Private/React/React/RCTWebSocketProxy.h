../../../../../../node_modules/react-native/React/Base/RCTWebSocketProxy.h