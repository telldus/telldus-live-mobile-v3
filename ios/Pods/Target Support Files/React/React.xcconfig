CLANG_CXX_LANGUAGE_STANDARD = c++14
CONFIGURATION_BUILD_DIR = $PODS_CONFIGURATION_BUILD_DIR/React
GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1
HEADER_SEARCH_PATHS = "${PODS_ROOT}/Headers/Private" "${PODS_ROOT}/Headers/Private/React" "${PODS_ROOT}/Headers/Public" "${PODS_ROOT}/Headers/Public/React"
OTHER_LDFLAGS = -l"stdc++" -framework "JavaScriptCore"
PODS_BUILD_DIR = $BUILD_DIR
PODS_CONFIGURATION_BUILD_DIR = $PODS_BUILD_DIR/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)
PODS_ROOT = ${SRCROOT}
PRODUCT_BUNDLE_IDENTIFIER = org.cocoapods.${PRODUCT_NAME:rfc1034identifier}
SKIP_INSTALL = YES
