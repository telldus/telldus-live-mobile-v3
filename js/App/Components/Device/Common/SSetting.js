/**
 * Copyright 2016-present Telldus Technologies AB.
 *
 * This file is part of the Telldus Live! app.
 *
 * Telldus Live! app is free : you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Telldus Live! app is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Telldus Live! app.  If not, see <http://www.gnu.org/licenses/>.
 *
 */

// @flow

'use strict';

import React from 'react';
import { useSelector } from 'react-redux';
import { RadioButtonInput } from 'react-native-simple-radio-button';

import {
	View,
	Text,
} from '../../../../BaseComponents';

import Theme from '../../../Theme';

const SSetting = (props: Object): Object => {
	const {
		index,
		value,
		onPressOne,
		onPressTwo,
		onPressThree,
		isOneSelected,
		isTwoSelected,
		isThreeSelected,
	} = props;

	const { layout } = useSelector((state: Object): Object => state.app);

	const {
		optionWrapStyle,
		optionButtonCover,
		brandPrimary,
		sRBCoverStyle,
		sRBHTextStyle,
		buttonOuterSizeV,
		buttonSizeV,
	} = getStyles(layout);


	return (
		<View style={optionButtonCover}>
			<View style={sRBCoverStyle}>
				{index === 0 && <Text style={sRBHTextStyle}>
					{1}
				</Text>
				}
				<RadioButtonInput
					isSelected={isOneSelected}
					buttonWrapStyle={optionWrapStyle}
					buttonOuterSize={buttonOuterSizeV}
					buttonSize={buttonSizeV}
					borderWidth={3}
					buttonInnerColor={brandPrimary}
					buttonOuterColor={brandPrimary}
					onPress={onPressOne}
					obj={{value}}
					index={index}/>
			</View>
			<View style={sRBCoverStyle}>
				{index === 0 && <Text style={sRBHTextStyle}>
					{'-'}
				</Text>
				}
				<RadioButtonInput
					isSelected={isTwoSelected}
					buttonWrapStyle={optionWrapStyle}
					buttonOuterSize={buttonOuterSizeV}
					buttonSize={buttonSizeV}
					borderWidth={3}
					buttonInnerColor={brandPrimary}
					buttonOuterColor={brandPrimary}
					onPress={onPressTwo}
					obj={{value}}
					index={index}/>
			</View>
			<View style={sRBCoverStyle}>
				{index === 0 && <Text style={sRBHTextStyle}>
					{0}
				</Text>
				}
				<RadioButtonInput
					isSelected={isThreeSelected}
					buttonWrapStyle={optionWrapStyle}
					buttonOuterSize={buttonOuterSizeV}
					buttonSize={buttonSizeV}
					borderWidth={3}
					buttonInnerColor={brandPrimary}
					buttonOuterColor={brandPrimary}
					onPress={onPressThree}
					obj={{value}}
					index={index}/>
			</View>
		</View>
	);

};

const getStyles = (appLayout: Object): Object => {
	const { height, width } = appLayout;
	const isPortrait = height > width;
	const deviceWidth = isPortrait ? width : height;

	const {
		paddingFactor,
		brandPrimary,
	} = Theme.Core;

	const padding = deviceWidth * paddingFactor;

	const fontSizeText = deviceWidth * 0.035;

	const outerPadding = padding * 2;

	const totalPaddingV = padding * 11;
	const buttonOuterSizeV = Math.floor((deviceWidth - (outerPadding + totalPaddingV)) / 10);
	const buttonSizeV = buttonOuterSizeV * 0.5;

	return {
		buttonOuterSizeV,
		buttonSizeV,
		brandPrimary,
		optionButtonCover: {
			flexDirection: 'column',
			alignItems: 'center',
			justifyContent: 'center',
		},
		optionWrapStyle: {
			marginTop: padding,
			marginLeft: padding,
		},
		sRBCoverStyle: {
			flexDirection: 'row',
			alignItems: 'center',
			justifyContent: 'center',
		},
		sRBHTextStyle: {
			marginTop: padding,
			marginLeft: padding,
			fontSize: fontSizeText * 1.2,
			borderColor: brandPrimary,
			textAlignVertical: 'center',
		},
	};
};

export default React.memo<Object>(SSetting);

