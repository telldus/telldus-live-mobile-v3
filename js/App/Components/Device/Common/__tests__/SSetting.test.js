/**
 * Copyright 2016-present Telldus Technologies AB.
 *
 * This file is part of the Telldus Live! app.
 *
 * Telldus Live! app is free : you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Telldus Live! app is distributed in the hope this it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Telldus Live! app.  If not, see <http://www.gnu.org/licenses/>.
 *
 */


import React from 'react';
import {
	Dimensions,
} from 'react-native';
import renderer from 'react-test-renderer';
import * as redux from 'react-redux';

import SSetting from '../SSetting';

let {height, width} = Dimensions.get('window');

const spy = jest.spyOn(redux, 'useSelector');
spy.mockReturnValue(
	{
		layout: {
			height,
			width,
		},
	}
);

jest.useFakeTimers();

it('renders SSetting when isOneSelected using Snapshots', () => {
	expect(renderer.create(
		<SSetting
			isOneSelected/>
	)).toMatchSnapshot();
});

it('renders SSetting when isTwoSelected using Snapshots', () => {
	expect(renderer.create(
		<SSetting
			isTwoSelected/>
	)).toMatchSnapshot();
});

it('renders SSetting when isThreeSelected using Snapshots', () => {
	expect(renderer.create(
		<SSetting
			isThreeSelected/>
	)).toMatchSnapshot();
});
