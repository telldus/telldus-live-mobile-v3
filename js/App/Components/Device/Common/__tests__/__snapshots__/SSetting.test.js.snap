// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SSetting when isOneSelected using Snapshots 1`] = `
<View
  colorScheme="light"
  colors={
    Object {
      "ShadeOne": "#eeeeee",
      "ShadeThree": "#8e8e93",
      "ShadeTwo": "#b5b5b5",
      "activeTintOne": "#e26901",
      "background": "#fff",
      "border": "#C9C9CE",
      "card": "#fff",
      "inActiveTintOne": "#e26901",
      "primary": "#1b365d",
      "screenBackground": "#eeeeee",
      "text": "#A59F9A",
      "textFive": "#555555",
      "textFour": "#4C4C4C",
      "textThree": "#000",
      "textTwo": "#777777",
    }
  }
  dark={false}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              Object {
                "backgroundColor": "#2196f3",
                "height": 20,
                "width": 20,
              },
              Object {
                "borderRadius": 11.75,
                "height": 23.5,
                "width": 23.5,
              },
              Object {
                "backgroundColor": "#1b365d",
              },
            ]
          }
        />
      </View>
    </View>
  </View>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              undefined,
              undefined,
              undefined,
            ]
          }
        />
      </View>
    </View>
  </View>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              undefined,
              undefined,
              undefined,
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`renders SSetting when isThreeSelected using Snapshots 1`] = `
<View
  colorScheme="light"
  colors={
    Object {
      "ShadeOne": "#eeeeee",
      "ShadeThree": "#8e8e93",
      "ShadeTwo": "#b5b5b5",
      "activeTintOne": "#e26901",
      "background": "#fff",
      "border": "#C9C9CE",
      "card": "#fff",
      "inActiveTintOne": "#e26901",
      "primary": "#1b365d",
      "screenBackground": "#eeeeee",
      "text": "#A59F9A",
      "textFive": "#555555",
      "textFour": "#4C4C4C",
      "textThree": "#000",
      "textTwo": "#777777",
    }
  }
  dark={false}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              undefined,
              undefined,
              undefined,
            ]
          }
        />
      </View>
    </View>
  </View>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              undefined,
              undefined,
              undefined,
            ]
          }
        />
      </View>
    </View>
  </View>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              Object {
                "backgroundColor": "#2196f3",
                "height": 20,
                "width": 20,
              },
              Object {
                "borderRadius": 11.75,
                "height": 23.5,
                "width": 23.5,
              },
              Object {
                "backgroundColor": "#1b365d",
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`renders SSetting when isTwoSelected using Snapshots 1`] = `
<View
  colorScheme="light"
  colors={
    Object {
      "ShadeOne": "#eeeeee",
      "ShadeThree": "#8e8e93",
      "ShadeTwo": "#b5b5b5",
      "activeTintOne": "#e26901",
      "background": "#fff",
      "border": "#C9C9CE",
      "card": "#fff",
      "inActiveTintOne": "#e26901",
      "primary": "#1b365d",
      "screenBackground": "#eeeeee",
      "text": "#A59F9A",
      "textFive": "#555555",
      "textFour": "#4C4C4C",
      "textThree": "#000",
      "textTwo": "#777777",
    }
  }
  dark={false}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              undefined,
              undefined,
              undefined,
            ]
          }
        />
      </View>
    </View>
  </View>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              Object {
                "backgroundColor": "#2196f3",
                "height": 20,
                "width": 20,
              },
              Object {
                "borderRadius": 11.75,
                "height": 23.5,
                "width": 23.5,
              },
              Object {
                "backgroundColor": "#1b365d",
              },
            ]
          }
        />
      </View>
    </View>
  </View>
  <View
    colorScheme="light"
    colors={
      Object {
        "ShadeOne": "#eeeeee",
        "ShadeThree": "#8e8e93",
        "ShadeTwo": "#b5b5b5",
        "activeTintOne": "#e26901",
        "background": "#fff",
        "border": "#C9C9CE",
        "card": "#fff",
        "inActiveTintOne": "#e26901",
        "primary": "#1b365d",
        "screenBackground": "#eeeeee",
        "text": "#A59F9A",
        "textFive": "#555555",
        "textFour": "#4C4C4C",
        "textThree": "#000",
        "textTwo": "#777777",
      }
    }
    dark={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 20.83275,
          "marginTop": 20.83275,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "borderColor": "#1b365d",
            "borderRadius": 23.5,
            "borderWidth": 3,
            "height": 47,
            "justifyContent": "center",
            "opacity": 1,
            "width": 47,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 10,
              },
              undefined,
              undefined,
              undefined,
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
